Sources/Basic/Path.swift

```diff
@@ -327,6 +327,51 @@ private struct PathImpl {
 }
 
 
+extension AbsolutePath {
+    /// Returns a relative path that, when concatenated to `base`, yields the
+    /// callee path itself.  If `base` is not an ancestor of the callee, the
+    /// returned path will begin with one or more `..` path components.
+    ///
+    /// Because both paths are absolute, they always have a common ancestor
+    /// (the root path, if nothing else).  Therefore, any path can be made
+    /// relative to any other path by using a sufficient number of `..` path
+    /// components.
+    ///
+    /// This method is strictly syntactic and does not access the file system
+    /// in any way.  Therefore, it does not take symbolic links into account.
+    public func relative(to base: AbsolutePath) -> RelativePath {
```

Comment:
Yes, that was a bit of hyperbole.  Strongly dislike, then, because the actual operation really seems more like appending a second thing to a first thing, whereas joining (in the sense in which our APIs use it) is to combine all the elements of a sequence.

I actually think that the word "append" iis more appropriate here, though that would of course be for a mutating method.

I definitely agree that an operator, if we have one, would be just an alias for whatever name we choose.